docs/config.json

```
@@ -920,7 +920,11 @@
           "slug": "/kubernetes-access/getting-started/"
         },
         {
-          "title": "Kubernetes Discovery",
+          "title": "Kubernetes Apps Discovery",
```

```suggestion
          "title": "Kubernetes App Discovery",
```
Making this singular as in "Service Discovery" or "Incident Response"

---

I did it on purpose, because we have a problem with name ambiguity when it singular. Now that we have both discovery **of** clusters and **inside** clusters, it's not clear which one "Kubernetes Cluster Discovery" means. So to make it more clear I used "Kubernetes Clusters Discovery" and using same pattern "Kubernetes Apps Discovery", though I don't think it's a hard requirement for the Apps, since we don't have this problem with ambiguity. Does it make sense?

---

I'm not sure I can gather the distinction between "of clusters" and "inside clusters" from the term "Kubernetes Clusters Discovery". To me, this term sounds like it means, "the discovery of Kubernetes clusters" in a way that clashes a bit with the conventions of written English (it should be "Kubernetes cluster discovery").

I would use another title to make this clearer.

---

Yep, "Kubernetes Clusters Discovery" means "the discovery of Kubernetes clusters" and I think it's clear. The problem is with "Kubernetes Cluster Discovery" which to me is not as clear (it might means discovery inside cluster).

I'm fine with changing it to how you suggested if you think it doesn't sound confusing.
So we'll have:

"Kubernetes Cluster Discovery" - discovery of Kubernetes clusters (in AWS, GCP etc)
"Kubernetes App Discovery"      - discovery of Apps inside a Kubernetes cluster.

---

```
"Kubernetes Cluster Discovery" - discovery of Kubernetes clusters (in AWS, GCP etc)
"Kubernetes App Discovery" - discovery of Apps inside a Kubernetes cluster.
```
This is how I understand it